contracts/tgrade-community-pool/src/msg.rs

```diff
@@ -2,7 +2,7 @@ use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
 use cosmwasm_std::Coin;
-use cw3::Vote;
+use tg3::Vote;
```

Comment:
Something seems odd to me here.

If these have different field names, I would expect some diff in this repo when I construct Vote to use `points` not `weight`.

Reply:
Most of things happens in voting contracts package. In contracts they are just passed around. Thanks to good code abstraction.